database/seeds/SponsorsSeeder.php

```diff
@@ -11,11 +12,93 @@ class SponsorsSeeder extends Seeder
      */
     public function run()
     {
-        //create RVNT sponsor for tests
-        $sponsor = ['name' => "Real Virtual Project", "shortcode" =>"RVNT"];
-        factory(App\Sponsor::class)->create($sponsor);
+        // create RVNT sponsor for tests
+        // also used by 3MAY20-VC1-CH6-HAS-112019
+        $sponsor = factory(App\Sponsor::class)->create(['name' => "Real Virtual Project", "shortcode" =>"RVNT"]);
+
+        $sponsor->evaluations()->saveMany($this->qualifyPrimarySchoolers());
+        $sponsor->evaluations()->saveMany($this->veryfiesKids());
 
         // And 5 default factory models to be able to mirror live data
         factory(App\Sponsor::class, 5)->create();
+
+        // Gets the SK rules
+        // used by 2MAY20-VC1-CH2-HI-122018
+        $sponsor2 = App\Sponsor::find(2);
+        $sponsor2->evaluations()->saveMany($this->qualifyPrimarySchoolers());
+        $sponsor2->evaluations()->saveMany($this->veryfiesKids());
+
+
+        // Gets SK rules, without the primary school disqualification
+        // used by 5MAY20-VC1-CH3-HA-112015
+        $sponsor3 = App\Sponsor::find(3);
+        $sponsor3->evaluations()->saveMany($this->allowPrimarySchoolers());
+        $sponsor3->evaluations()->saveMany($this->veryfiesKids());
+
+        // only gets verifications
+        // used by 6MAY-20-VC1-CH2-032020
+        $sponsor4 = App\Sponsor::find(4);
+        $sponsor4->evaluations()->saveMany($this->veryfiesKids());
+
+        // 4MAY20-VC1-CH1P-HA-012020 is in sponsor 5, unmodified
+        // 1MAY20a-VC2-CH1-HI-042019 is in sponsor 5, unmodified
+    }
+
+    public function veryfiesKids()
+    {
+        return [
+            new Evaluation([
+                "name" => "FamilyHasUnverifiedChildren",
+                "value" => 0,
+                "purpose" => "notices",
+                "entity" => "App\Family",
+            ])
+        ];
+    }
+
+    public function allowPrimarySchoolers()
```

Comment:
Would these be useful utility functions in for console command? Will be fiddly setting up new sponsors without such tooling.